crates/bevy_color/src/linear_rgba.rs

```diff
@@ -205,6 +228,17 @@ impl From<Hsla> for LinearRgba {
     }
 }
 
+impl From<LinearRgba> for wgpu::Color {
+    fn from(color: LinearRgba) -> Self {
+        wgpu::Color {
+            r: color.red as f64,
+            g: color.green as f64,
+            b: color.blue as f64,
+            a: color.alpha as f64,
+        }
+    }
+}
```

Comment:
Since `bevy_color` is our new color management library specifically for Bevy, would it be worth moving to `f64` internally to avoid this casting? The extra precision will eat up more memory, but also expose the full precision of `wgpu` to end users.

Reply:
That's a good question. I think that should be investigated after migration: then we can get both benchmark results and see if there's meaningfully improved color fidelity or numerical stability.

Reply:
Probably won't make a big difference on the code-path where `wgpu` [almost immediately casts the color back to a `f32`](https://github.com/gfx-rs/wgpu/blob/a3159258824cb7faeaac74a32f46ffe0aa4fe3e2/wgpu-core/src/command/render.rs#L1787) :smile:

Reply:
Well, at this rate I guess we're getting `bevy_wgpu`